client/gutenberg/extensions/tiled-gallery/layout/mosaic/resize.js

```diff
@@ -41,7 +41,7 @@ function getRowCols( row ) {
 }
 
 function getColImgs( col ) {
-	return Array.from( col.querySelectorAll( '.tiled-gallery__item > img' ) );
+	return Array.from( col.querySelectorAll( '.tiled-gallery__item img' ) );
```

Comment:
What do you think of 

```suggestion
	return Array.from( col.querySelectorAll( '.tiled-gallery__item > img, .tiled-gallery__item > a > img' ) );
```

The `img` descendent of concerns me a bit when putting a `RichText` caption somewhere in there.

Reply:
Valid point and your suggestion tests well. 👍